### PR TITLE
Ot153 112 members reducer

### PR DESCRIPTION
**Pull request by frangdelsolar**
## JIRA
[OT153-112](https://alkemy-labs.atlassian.net/browse/OT153-112)

## Short description

Implemented redux for members component

## Screenshot or Video

image

## Teammate collaborator

- [ x] Someone from the team collaborated (**_if so, please tag them after this line_**).
- @NLuna19 
Nico helped me debugging some issues with my redux store. 

### Sanity Check Steps

- [x] All console logs have been removed.
- [x] This change was tested on my localhost.
- [ ] This change depends for another ticket.
- [x] All linked JIRA acceptance criteria are satisfied.
- [ ] Does this change require environment variables.

### CODE REVIEW

File: src/app/features/public/pages/about/components/us/us.component.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ import { HttpService } from '@app/core/services/http.service';
   styleUrls: ['./us.component.scss']
 })
 export class UsComponent implements OnInit {
-  members  : Members[];
+  members  : Member[];
   url = 'http://ongapi.alkemy.org/api/members';
```
Comment (RicardoMaldonado93) [inline src/app/features/public/pages/about/components/us/us.component.ts]: cambiar esto por una variable de entorno

**Review by RicardoMaldonado93 — CHANGES_REQUESTED** [review]
Please fix the observation to continue with the PR

**Comment by frangdelsolar** [comment]
@RicardoMaldonado93 issue fixed

**Review by RicardoMaldonado93 — APPROVED** [review]
Good job!

**Pull request merged**